src/site/content/en/fast/cookie-notice-best-practices/index.md

```diff
@@ -0,0 +1,445 @@
+performance data.
+
+Sites should be aware of this phenomenon; it is also worthwhile to understand
+the mechanisms that your RUM tooling uses to collect its data. However, for the
+typical site this discrepancy probably isn't a cause for alarm given the
+direction and magnitude of the data skew. Cookie usage is not a technical
+requirement for performance measurement. The
+[web-vitals](https://github.com/GoogleChrome/web-vitals) JavaScript library is
+an example of a library that does not use cookies.
+
+It's also worth noting that depending on how your site uses cookies to collect
+performance data (that is, whether the cookies contain personal information), as
+well as the legislation in question, the use of cookies for performance
+measurement might not be subject to the same legislative requirements as some of
+the cookies used on your site for other purposes—for example, advertising
+cookies. Some sites choose to break out performance cookies as a separate
+category of cookies when asking for user consent.
+
+
+### Synthetic monitoring
+
+
+Without custom configuration, most synthetic tools will only measure the
+experience of a first-time user who has not responded to a cookie consent
+notice. However, not only do variations in cache state (for example, an initial
+visit versus a repeat visit) need to be considered when collection performance
+data, but also variations in cookie acceptance state—accepted, rejected, or
+unresponded.
+
+
+The following sections discuss WebPageTest and Lighthouse settings that can be
+helpful for incorporating cookie notices into performance measurement workflows.
+However, cookies and cookie notices are just one of many factors that can be
+difficult to perfectly simulate in lab environments. For this reason, it is
+important to make [RUM
+data](https://web.dev/user-centric-performance-metrics/#how-metrics-are-measured)
+the cornerstone of your performance benchmarking, rather than synthetic tooling.
+
+
+### Testing cookie notices with WebPageTest
+
+#### Scripting
+
+You can use scripting to have a [WebPageTest](https://webpagetest.org/) "click"
+the cookie consent banner while collecting a trace.
+
+Add a script by going to the **Script** tab. The script below navigates to the URL
+to be tested and then clicks the DOM element with the id `cookieButton`. 
+
+```shell
+combineSteps
+navigate    %URL%
+clickAndWait    id=cookieButton
+```
+
+When using this script be aware that:
+
+*   `combineSteps` tells WebPageTest to "combine" the results of the scripting
+    steps that follow into a single set of traces and measurements. Running this
+    script without `combineSteps` can also be useful—separate traces make it
+    easy to see whether resources were loaded before or after cookie acceptance.
+*   `%URL%` is a WebPageTest convention that refers to the URL that is being
+    tested.
+*   clickAndWait follows the format `clickAndWait attribute=Value`.
```

Comment:
```suggestion
*   `clickAndWait` follows the format `clickAndWait attribute=Value`.
```

Reply:
Can you expand a bit on `clickAndWait`? Something along the lines "tells WPT to click on an element specified with `attribute=value` and wait X amount of time before collecting the measurements"